app/src/main/java/org/mozilla/focus/fragment/FirstrunFragment.kt

```diff
@@ -62,6 +64,8 @@ class FirstrunFragment : Fragment(), View.OnClickListener {
             R.id.skip -> {
                 finishFirstrun()
                 Onboarding.skipButtonTapped.record(Onboarding.SkipButtonTappedExtra(currentItem))
+                TelemetryWrapper.skipFirstRunEvent()
+                TelemetryWrapper.finishFirstRunEvent()
```

Comment:
Shouldn't finish be below for `R.id.finish`?

Reply:
Good catch! Thank you!